pymanifold/lib.py

```diff
@@ -1,4 +1,6 @@
 import requests
+from numpy import log as ln, argmax
```

Comment:
Can you add numpy to the project's requirements? https://github.com/bcongdon/PyManifold/blob/main/pyproject.toml